### ISSUE TITLE
Fill option for inputs

### ISSUE THREAD

**Issue by silviyaboteva** [telerik/kendo-themes#1939]
**Is your feature request related to a problem? Please describe.**
Currently, the TextArea component design follows the **filled** text fields design (https://material.io/components/text-fields#anatomy). However, the design differs from the other input editors such as input, numerictextbox, maskedtextbox etc. as they follow the **flat** text fields design (https://github.com/telerik/kendo-themes/blob/develop/tests/visual/output/material/inputs.png). And this can be considered as inconsistency between our components' look and feel.

**Describe the solution you would like to see implemented**
We can base on the standardizing issue (https://github.com/telerik/k2/issues/241) and add a **fill** option to the input editors to set the different style options available. Check this link for reference: https://material-ui.com/components/text-fields/.

**Additional context**
The question was also raised from a client in ticket ID: 1487174

**Comment by eeasss** [comment]
Should we also include outline option? I think that most implementations of Material I've seen (on google cloud and angular material) also apply fill and outline to the dropdowns as well.

**Comment by silviyaboteva** [comment]
@eeasss yes we will. The `fill` will be the name of the property which will accept all available appearance values such as default (flat), solid and outline.
